CHANGES.rst

```diff
@@ -1,39 +1,61 @@
 Changes
 =======
 
-Version 0.5.0 (2019-04-24)
+Version 0.6.0 (2019-12-27)
 --------------------------
 
 - Users:
-
-  - Allows to explore workflow results by running interactive Jupyter notebook
-    sessions on the workspace files.
-  - Allows to declare computing resources needed for workflow runs, such as
-    access to CVMFS repositories.
-  - Improves ``reana-client`` command-line client with new options to stop
-    workflows, diff workflows, move and remove files.
-  - Upgrades CWL engine to 1.0.20181118133959.
-  - See additional changes in `reana-client release notes <https://reana-client.readthedocs.io/en/latest/changes.html#version-0-5-0-2019-04-24>`_.
+    - Adds support for HTCondor compute backends for all workflow engines.
```

Comment:
Maybe `workflow engine` is a bit too much our jargon? What do you think about something more neutral like this or similar:

"Adds [HTCondor](https://research.cs.wisc.edu/htcondor/) support for all workflow languages."

Reply:
Reformulated adding the workflow names at the end.